issue_reports.gemspec

```
@@ -0,0 +1,16 @@
+Gem::Specification.new do |s|
+  s.name = "issue_reports"
+  s.version = "1.0.0"
+  s.summary = ""
+  s.description = ""
+  s.authors = ["HeadLight Solutions"]
+  s.email = "support@headlightsolutions.com"
```

Just incase its gets spammy:

```suggestion
  s.email = "contact@headlightsolutions.com"
```